### DISCUSSION TITLE
Not able to view new primary categories while project creation dock-staging server

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-coKreat/Community#59] [Bugs]
While trying to create new project in dock-staging server, At project scope section in content type drop down **Observation , Survey** options are not coming, Even though these primary category definitions are added in staging. This `https://dockstaging.sunbirded.org/learner/channel/v1/read/01345815127107174426` URL is to get primary and additional primary categories.

Can somebody help on this ?
CC : @kiranharidas187

**Comment by Aparna-karri591** [comment]
@ramkumar-pacewisdom Our Team is looking into this.. will get back to you

**Reply by ramkumar-ns-dev** [reply]
@Aparna-karri591 okay

**Comment by vaivk369** [comment]
Can you please confirm if the categories are added against the channel are public?

**Reply by ramkumar-ns-dev** [reply]
@vaivk369 Sorry i didn't know about "channel are public" can you elaborate if it's possible ?

**Reply by vaivk369** [reply]
When you create the object categories the type of the category should be public and the channel to be set to specific category,
